<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saleman</class>
 <widget class="QMainWindow" name="saleman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_cashier">
     <attribute name="title">
      <string>收银</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_cashier">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>501</width>
        <height>221</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <column>
       <property name="text">
        <string>商品号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>商品名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>单价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>金额</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_cashier_settle">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>390</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>结账</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cashier_delete">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>390</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_gno">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_amount">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_total_money">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>金额总计：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_cashier_amount">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cashier_gno">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_cashier_total_money">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>220</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cashier_add">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_goods_amount">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>170</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_goodsamount">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>商品件数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_cno">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>300</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>会员卡号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cashier_cno">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>310</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_usecard">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>270</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>     使用会员卡</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_cmonry">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>330</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>卡内余额：</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_cashier_cmoney">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>340</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_all2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>420</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>会员卡总计：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_all1">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>420</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>现金总计：</string>
      </property>
     </widget>
     <zorder>tableWidget_cashier</zorder>
     <zorder>pushButton_cashier_settle</zorder>
     <zorder>pushButton_cashier_delete</zorder>
     <zorder>label_cashier_gno</zorder>
     <zorder>label_amount</zorder>
     <zorder>label_cashier_total_money</zorder>
     <zorder>spinBox_cashier_amount</zorder>
     <zorder>lineEdit_cashier_gno</zorder>
     <zorder>textBrowser_cashier_total_money</zorder>
     <zorder>pushButton_cashier_add</zorder>
     <zorder>textBrowser_goods_amount</zorder>
     <zorder>label_cashier_goodsamount</zorder>
     <zorder>label_cashier_cno</zorder>
     <zorder>lineEdit_cashier_cno</zorder>
     <zorder>radioButton_usecard</zorder>
     <zorder>label_cashier_cmonry</zorder>
     <zorder>textBrowser_cashier_cmoney</zorder>
     <zorder>label_cashier_all1</zorder>
     <zorder>label_cashier_all2</zorder>
     <zorder>label_cashier_all1</zorder>
    </widget>
    <widget class="QWidget" name="tab_card">
     <attribute name="title">
      <string>办卡</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>电话</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>金额</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>有效期至</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_phone">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_money">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_date">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>卡号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cno">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_cashier_all3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>410</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>办卡金额总计：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_cashier_exit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>470</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cashier_settle_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改个人信息</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>pushButton_cashier_exit</zorder>
   <zorder>pushButton_cashier_settle_2</zorder>
   <zorder>textBrowser_goods_amount_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
